link/xdp_test.go

```diff
@@ -0,0 +1,45 @@
+package link
+
+import (
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/asm"
+	"github.com/cilium/ebpf/internal/testutils"
+)
+
+func TestAttachXDP(t *testing.T) {
+	testutils.SkipOnOldKernel(t, "5.7", "BPF_LINK_TYPE_XDP")
+	prog, err := ebpf.NewProgram(&ebpf.ProgramSpec{
+		Type: ebpf.XDP,
+		Instructions: asm.Instructions{
+			asm.LoadImm(asm.R0, 2, asm.DWord),
+			asm.Return(),
+		},
+		License: "GPL",
```

Comment:
MIT?

Reply:
I think GPL for XDP without using any specific map. should I change it to MIT or Dual MIT/GPL?

Reply:
I prefer MIT if there is no strong reason to use GPL (such as a helper call) since the repo is MIT as well.

Reply:
Updated.